src/app/pages/send-tokens/components/asset-search/asset-search.tsx

```diff
@@ -142,6 +136,9 @@ const AssetSearchField = memo(
             onChange={(e: FormEvent<HTMLInputElement>) => {
               const { value } = e.currentTarget;
               setSearchInput(value);
+              setAssetItems(
+                assets.filter(asset => asset.name.toLowerCase().includes(value.toLowerCase()))
```

Comment:
Could this logic be extracted to a fn with descriptive name?

Reply:
Yep, I can.